tf/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "aws_region" {
+  type        = string
+  description = "aws region"
+  default     = "eu-central-1"
+}
+
+variable "log_level" {
+  type        = string
+  description = "lambda extension log level"
+  default     = "trace"
+}
+
+variable "ec_region" {
+  type        = string
+  description = "ec region"
+  default     = "aws-eu-central-1"
+}
+
+variable "ec_deployment_template" {
```

Comment:
Do we need this? Maybe start without it?

Reply:
This is required to fully support custom region. Different regions have different templates (there are some regions with common templates however)